src/inputs/input/PTextInput.vue

```diff
@@ -41,13 +41,22 @@
                     {{ value }}
                 </slot>
             </span>
-            <p-i
-                v-show="(isFocused || isInvalid)"
-                class="delete-all-icon"
-                name="ic_delete" height="1rem" width="1rem"
-                color="inherit transparent"
-                @mousedown.native.prevent
-                @click="handleDeleteAllTags"
+            <p-button v-if="($attrs.type === 'password') && visiblePassword"
```

Comment:
```suggestion
            <p-button v-if="(inputType === 'password') && visiblePassword"
```

I think it would be much better to use refined state `inputType`.